force-app/tests/LDV Recipes/LDVRecipes_Tests.cls

```diff
@@ -0,0 +1,36 @@
+@isTest
+private class LDVRecipes_Tests {
+    @isTest
+    static void testLDVRecipesQueueable() {
+        StaticResource testData = [
+            SELECT Id, Body, Name
+            FROM StaticResource
+            WHERE Name IN ('helloworld')
+            LIMIT 1
+        ];
+        TriggerHandler.bypass('AccountTriggerHandler');
+        List<Account> accounts = (List<Account>) TestFactory.createSObjectList(
+            new Account(),
+            20,
+            true
+        );
+        for (Account account : accounts) {
+            FilesRecipes.createFileAttachedToRecord(
+                testData.Body,
+                account.Id,
+                'helloWorld.docx'
+            );
+        }
+
+        Test.startTest();
+        LDVRecipes ldvr = new LDVRecipes();
+        System.enqueueJob(ldvr);
+        Test.stopTest();
+
+        System.assertEquals(
+            1,
+            LDVRecipes.chunksExecuted,
+            'Expected to find the queueable executed 4 times'
```

Comment:
Pretty confusing messgae! Says 4 times executed while asserts 1.

Reply:
Good catch. This is an artifact of me trying to get re-entrant async tests working. (hint: they don't.)